WordPressAuthenticator.xcodeproj/project.pbxproj

```diff
@@ -587,6 +589,8 @@
 		3FE8072329365FC20088420C /* GoogleSignIn */ = {
 			isa = PBXGroup;
 			children = (
+				3F30A6B9299F12E30004452F /* Character+URLSafeTests.swift */,
```

Comment:
This addition is because I sorted the group alphabetically. See the matching removal below.